keyless/pepper/service/src/main.rs

```diff
@@ -153,7 +159,7 @@ async fn main() {
         };
 
     // Start the JWK fetchers
-    let jwk_cache = jwk_fetcher::start_jwk_fetchers();
+    let jwk_cache = jwk_fetcher::start_jwk_fetchers(args.jwk_issuers_override.clone());
```

Comment:
nit: does `jwk_issuers_override` make sense as a name for this argument? It sounds like it's overriding the defaults, as opposed to being added to the set. Perhaps something like `additional_jwk_issuers`? 😄